module-ballerina-wsdl/wsdl_tool.bal

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2024 WSO2 Inc. (http://www.wso2.org).
```

Comment:
We don't need any Ballerina code inside our package. We can have empty projects when we do a `bal pack` for tools.